Extension/ui/settings.ts

```diff
@@ -95,26 +107,50 @@ class SettingsApp {
     private updateErrors(errors: any): void {
         this.updating = true;
         try {
-            this.showErrorWithInfo(elementId.intelliSenseModeInvalid, 
-                    errors.intelliSenseMode ? true : false, 
-                    errors.intelliSenseMode);
-
-            this.showErrorWithInfo(elementId.compilerPathInvalid, 
-                    errors.compilerPath ? true : false, 
-                    errors.compilerPath);
-
-            this.showErrorWithInfo(elementId.includePathInvalid, 
-                    errors.includePath ? true : false, 
-                    errors.includePath);
+            this.showErrorWithInfo(elementId.intelliSenseModeInvalid, errors.intelliSenseMode);
+            this.showErrorWithInfo(elementId.compilerPathInvalid, errors.compilerPath);
+            this.showErrorWithInfo(elementId.includePathInvalid, errors.includePath);
         } finally {
             this.updating = false;
         }
     }
 
-    private showErrorWithInfo(elementID: string, show: boolean, errorInfo: string): void {
-         document.getElementById(elementID).style.visibility = show ? "visible" : "hidden";
+    private showErrorWithInfo(elementID: string, errorInfo: string): void {
+         document.getElementById(elementID).style.visibility = errorInfo ? "visible" : "hidden";
          document.getElementById(elementID).innerHTML = errorInfo ? errorInfo : "";
     }
+
+    private setKnownCompilers(compilers: string[]): void {
+        let list: HTMLElement = document.getElementById(elementId.knownCompilers);
+
+        // No need to add items unless webview is reloaded, in which case it will not have any elements.
+        // Otherwise, add items again.
+        if (list.firstChild) {
+           return;
+        }
+
+        if (compilers.length === 0) {
+            const noCompilers: string = "(No other compiler paths detected)";

```

Comment:
Why is the word "other" used? Can that be removed?

Reply:
"other" was used in comparison to a specified path. This doesn't make sense though to use just for the list of detected paths. 

Changing to "(No compiler paths detected)"